agent/src/main/java/com/teamscale/jacoco/agent/testimpact/TestEventHandlerStrategyBase.java

```diff
@@ -90,7 +90,11 @@ public String testRunStart(List<ClusteredTestDetails> availableTests, boolean in
 					" Thus, you cannot use the agent to retrieve impacted tests via the testrun/start REST endpoint." +
 					" Please use the 'teamscale-' agent parameters to configure a Teamscale connection.");
 		}
-
+		if (agentOptions.getTeamscaleServerOptions().commit == null) {
+			throw new UnsupportedOperationException("You did not provide a 'teamscale-commit' or " +
+					"'teamscale-commit-manifest-jar'. 'teamscale-revision' is not sufficient to retrieve impacted " +
```

Comment:
I found it a bit odd that `AgentOptions` doesn't contain string constants for the various options, given that they are included in a lot of error messages (which is very helpful!).

Reply:
Started adding constants for it.